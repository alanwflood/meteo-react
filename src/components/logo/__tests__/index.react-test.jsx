import React from "react";
import renderer from "react-test-renderer";
import Logo from "../index";

test("Logo Renders", () => {
  const component = renderer.create(<Logo />);

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
