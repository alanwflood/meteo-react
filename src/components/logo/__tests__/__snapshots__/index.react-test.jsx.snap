// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Renders 1`] = `
<div
  className="logo"
>
  <h1>
    <div>
      Meteo
    </div>
    <small>
      Weather at a Glance
    </small>
  </h1>
  <div
    className="tagline"
  >
    Search for a location above to get your 5 day weather forecast!
  </div>
</div>
`;
